testing/schemaPath1/test_ts.ts

```diff
@@ -9,12 +9,18 @@ const typeDefs = gql`
 const resolvers = {
 	Query: {
 		test_ts: function() {
-			return true;
+			const wut = 2
+			 *
+			2;
+
+					const badLint = "something";

```

Comment:
Codacy has a fix for the issue: [Expected indentation of 3 tabs but found 5.](https://app.codacy.com/gh/simpleviewinc/sv-graphql-client/pullRequest?prid=9144006)
```suggestion
			const badLint = "something";
```